dashboard/app/models/levels/level.rb

```diff
@@ -103,8 +103,7 @@ def related_videos
   end
 
   def specified_autoplay_video
-    @@specified_autoplay_video ||= {}
-    @@specified_autoplay_video[video_key] ||= Video.current_locale.find_by_key(video_key) unless video_key.nil?
+    Video.current_locale.find_by_key(video_key)
```

Comment:
Should we still be caching this DB lookup, but also keyed by locale?

Reply:
Done.